drivers/Ticker.cpp

```diff
@@ -22,6 +22,12 @@
 
 namespace mbed {
 
+void Ticker::~Ticker() {
```

Comment:
This should probably call detach to ensure no more interrupts can fire.

Reply:
👍